tmt/convert.py

```diff
@@ -389,6 +413,7 @@ def read(
             restraint_file = True
             echo(style('Restraint ', fg='blue'), nl=False)
```

Comment:
@ukulekek maybe it would be better to add an else condition here and raise some resonable error, instead of the assert below. The thing is, if somebody does not pass required variables to this function, I would rather like to see a nice error than a assertion error. Seems to me a clear and unhandled error condition. WDYT @ukulekek

Reply:
Thanks, error is added